src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -2024,10 +2015,6 @@ in storage.js
 	padding-bottom: 0;
 }
 
-.scraped-example:not(.expanded) .code-wrapper pre.src-line-numbers {
```

Comment:
I wonder why it was hidden until now. In your test doc file it works just as expected though so all good.